src/loaders/express/helmet.ts

```diff
@@ -73,10 +74,22 @@ const helmetMiddlewares = () => {
         'https://www.gstatic.cn/',
       ],
       formAction: ["'self'"],
-      upgradeInsecureRequests: !config.isDev,
-      reportUri: config.cspReportUri,
     },
-  })
+  }
+
+  // Add on upgradeInsecureRequest CSP header if !config.isDev
+  if (!config.isDev) {
+    set(cspDirectives, 'directives.upgradeInsecureRequests', [])
```

Comment:
could you add a note that an empty array still sets the upgrade-insecure-requests header? and that this key needs to be excluded in order for the header to be excluded (i.e. a falsey value doesn’t work)?

Reply:
also link to the helmet docs

Reply:
Added

Reply:
very nice